packages/terra-time-input/src/TimeUtil.js

```diff
@@ -269,6 +288,141 @@ class TimeUtil {
 
     return variant;
   }
+
+  /**
+   * Is the Time Input operating as a 12-hour variant?
+   * @param {Object} The TimeInput props.
+   * @returns {Boolean} True iff the locale allows a 12-hour time and the variant is 12-hour time.
+   */
+  static is12Hour(props) {
+    return TimeUtil.getVariantFromLocale(props) === TimeUtil.FORMAT_12_HOUR;
+  }
+
+  /**
+   * Is the Time Input's state representing a 12-hour time in the PM?
+   * @param {Object} props The TimeInput props.
+   * @param {Object} state The TimeInput state.
+   * @param {String} postMeridiem The PM string to compare against.
+   * @returns {Boolean} True iff the time input is in 12-hour mode and the state is a PM time.
```

Comment:
```suggestion
   * @returns {Boolean} True if the time input is in 12-hour mode and the state is a PM time.
```

Reply:
Like above, _iff_ meaning _if and only if_.